bc_obps/common/fixtures/dashboard/administration/internal_admin.json

```diff
@@ -5,7 +5,12 @@
       "name": "Internal Admin Administration Dashboard",
```

Comment:
I think we ultimately won't need this file because all the cas roles have at least view only access to everything. I'll clean it up in the cas_admin/cas_admin card